examples/graph.cc

```diff
@@ -0,0 +1,44 @@
+#include "dag/DirectedAcyclicGraph.h"
+
+/** @file graph.cc
+  *
+  * This is a very simple example for using the nodes and visitors defined by DirectedAcyclicGraph.
+  * The idpair is very similar to the PODIO ObjectID, so this may be instructive. For a more complete
+  * example on how to use it in the context of the fcc-edm, have a look at the read example in fcc-physics.
```

Comment:
I may have missed something, but the read example in fcc-physics doesn't use graphs?